% Options for packages loaded elsewhere
\PassOptionsToPackage{unicode}{hyperref}
\PassOptionsToPackage{hyphens}{url}
%
\documentclass[
  12pt,
]{article}
\usepackage{lmodern}
\usepackage{amssymb,amsmath}
\usepackage{ifxetex,ifluatex}
\ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
  \usepackage{textcomp} % provide euro and other symbols
\else % if luatex or xetex
  \usepackage{unicode-math}
  \defaultfontfeatures{Scale=MatchLowercase}
  \defaultfontfeatures[\rmfamily]{Ligatures=TeX,Scale=1}
\fi
% Use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
\IfFileExists{microtype.sty}{% use microtype if available
  \usepackage[]{microtype}
  \UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\makeatletter
\@ifundefined{KOMAClassName}{% if non-KOMA class
  \IfFileExists{parskip.sty}{%
    \usepackage{parskip}
  }{% else
    \setlength{\parindent}{0pt}
    \setlength{\parskip}{6pt plus 2pt minus 1pt}}
}{% if KOMA class
  \KOMAoptions{parskip=half}}
\makeatother
\usepackage{xcolor}
\IfFileExists{xurl.sty}{\usepackage{xurl}}{} % add URL line breaks if available
\IfFileExists{bookmark.sty}{\usepackage{bookmark}}{\usepackage{hyperref}}
\hypersetup{
  hidelinks,
  pdfcreator={LaTeX via pandoc}}
\urlstyle{same} % disable monospaced font for URLs
\usepackage[margin=1in]{geometry}
\usepackage{graphicx}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
% Set default figure placement to htbp
\makeatletter
\def\fps@figure{htbp}
\makeatother
\setlength{\emergencystretch}{3em} % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{-\maxdimen} % remove section numbering
\usepackage{helvet} % Helvetica font
\renewcommand*\familydefault{\sfdefault} % Use the sans serif version of the font
\usepackage[T1]{fontenc}

\usepackage[none]{hyphenat}

\usepackage{setspace}
\doublespacing
\setlength{\parskip}{1em}

\usepackage{lineno}

\usepackage{pdfpages}
\newlength{\cslhangindent}
\setlength{\cslhangindent}{1.5em}
\newenvironment{cslreferences}%
  {}%
  {\par}

\author{}
\date{\vspace{-2.5em}}

\begin{document}

\pagenumbering{arabic}
\linenumbers
\doublespacing

\hypertarget{diluted-fecal-community-transplant-restores-clostridioides-difficile-colonization-resistance-to-antibiotic-perturbed-murine-communities}{%
\section{\texorpdfstring{Diluted fecal community transplant restores
\emph{Clostridioides difficile} colonization resistance to antibiotic
perturbed murine
communities}{Diluted fecal community transplant restores Clostridioides difficile colonization resistance to antibiotic perturbed murine communities}}\label{diluted-fecal-community-transplant-restores-clostridioides-difficile-colonization-resistance-to-antibiotic-perturbed-murine-communities}}

\vspace{30mm}

\textbf{Running title:} Diluted fecal community transplant inhibits CDI

\vspace{20mm}

Nicholas A. Lesniak\textsuperscript{1}, Sarah
Tomkovich\textsuperscript{1}, Andrew Henry\textsuperscript{1}, Ana
Taylor\textsuperscript{1}, Joanna Colovas\textsuperscript{1}, Lucas
Bishop\textsuperscript{1}, Kathryn McBride\textsuperscript{1}, Patrick
D. Schloss\textsuperscript{1,\(\dagger\)}

\vspace{40mm}

\(\dagger\) To whom correspondence should be addressed:
\href{mailto:pschloss@umich.edu}{\nolinkurl{pschloss@umich.edu}}

1. Department of Microbiology and Immunology, University of Michigan,
Ann Arbor, MI 48109

\newpage

\hypertarget{abstract}{%
\subsection{Abstract}\label{abstract}}

Fecal communities transplanted into individuals can eliminate recurrent
\emph{Clostridioides difficile} infection (CDI) with high efficacy.
However, this treatment is only used once CDI becomes resistant to
antibiotics or has recurred multiple times. We sought to investigate
whether a fecal community transplant (FCT) pre-treatment could be used
to prevent CDI altogether. We treated male C57BL/6 mice with either
clindamycin, cefoperazone, or streptomycin, and then inoculated them
with the microbial community from untreated mice before challenging with
\emph{C. difficile}. We measured colonization and sequenced the V4
region of the 16S rRNA gene to understand the dynamics of the murine
fecal community in response to the FCT and \emph{C. difficile}
challenge. Clindamycin-treated mice became colonized with \emph{C.
difficile} but cleared it naturally and did not benefit from the FCT.
Cefoperazone-treated mice became colonized by \emph{C. difficile}, but
the FCT enabled clearance of \emph{C. difficile}. In
streptomycin-treated mice, the FCT was able to prevent \emph{C.
difficile} from colonizing. Then we diluted the FCT and repeated the
experiments. Cefoperazone-treated mice no longer cleared \emph{C.
difficile}. However, streptomycin-treated mice colonized with
1:10\textsuperscript{2} dilutions resisted \emph{C. difficile}
colonization. Streptomycin-treated mice that received a FCT diluted
1:10\textsuperscript{3}, \emph{C. difficile} colonized but later was
cleared. In streptomycin-treated mice, inhibition of \emph{C. difficile}
was associated with increased relative abundance of a group of bacteria
related to \emph{Porphyromonadaceae} and \emph{Lachnospiraceae}. These
data demonstrate that \emph{C. difficile} colonization resistance can be
restored to a susceptible community with a FCT as long as it complements
the missing populations.

\hypertarget{importance}{%
\subsection{Importance}\label{importance}}

Antibiotic use, ubiquitous with the healthcare environment, is a major
risk factor for \emph{Clostridioides difficile} infection (CDI), the
most common nosocomial infection. When \emph{C. difficile} becomes
resistant to antibiotics, a fecal microbiota transplant from a healthy
individual can effectively restore the gut bacterial community and
eliminate the infection. While this relationship between the gut
bacteria and CDI is well established, there are no therapies to treat a
perturbed gut community to prevent CDI. This study explored the
potential of restoring colonization resistance to antibiotic-induced
susceptible gut communities. We described the effect gut bacteria
community variation has on the effectiveness of a fecal community
transplant for inhibiting CDI. These data demonstrated that communities
susceptible to CDI can be supplemented with fecal communities but the
effectiveness depended on the structure of the community following the
perturbation. Thus, a reduced bacterial community may be able to recover
colonization resistance to patients treated with antibiotics.

\newpage

\hypertarget{introduction}{%
\subsection{Introduction}\label{introduction}}

The process by which gut bacteria prevent \emph{Clostridioides
difficile} and other pathogens from infecting and persisting in the
intestine is known as colonization resistance (1). Antibiotic-induced
disruption of the gut bacterial community breaks down colonization
resistance and is a major risk factor for \emph{C. difficile} infection
(CDI) (2). Gut bacteria inhibit \emph{C. difficile} through the
production of bacteriocins, modulation of available bile acids,
competition for nutrients, production of short-chain fatty acids, and
altering the integrity of the mucus layer (1). After the initial CDI is
cleared via antibiotics, patients can become reinfected. When CDI recurs
more than once, the gut bacterial community from a healthy person
typically is used to restore the gut community in the patient with
recurrent CDI (3). Fecal microbiota transplant (FMT) is effective, but
10 - 20\% of people that receive a FMT will still have another CDI (4).
Additionally, transfer of a whole fecal community can incidentally
transfer pathogens and cause adverse outcomes (5). While the FMT is
effective at curing recurrent CDI, it also has risks that must be
considered.

The benefits and risks of the FMT has led to the development of reduced
bacterial communities to treat CDI. Synthetic communities are more
defined than an FMT, making them easier to regulate as a drug. Tvede and
Rask-Madsen were the first to successfully treat CDI with a community of
isolates cultured from human feces (6). More recently, Lawley \emph{et
al.} analyzed murine experiments and the fecal communities from patients
with CDI to develop a synthetic community of six isolates to inhibit
\emph{C. difficile} colonization (7). Reduced communities derived from
human fecal communities, by methods such as selective isolation of
spores or culturing bacteria, have cured recurrent CDI in their initial
application (8--10). Although, a recent phase 2 trial of SER-109, a
spore-based treatment, failed its phase 2 clinical trial (11), these
therapies have the potential to offer the benefits of the FMT without
the associated risks but are only used once a patient has had multiple
CDIs. For these to be successful, we need a better approach to identify
candidate bacterial populations. Recently, an autologous FMT was shown
to be effective at restoring the gut microbiota in allogeneic
hematopoietic stem cell transplantation patients and prevented future
complications, such as systemic infections (12). It is unclear whether
an treatment similar to an autologous FMT or reduced bacterial
communities could be used to restore susceptible communities and prevent
CDI (13).

Because FMT is often sufficient to restore colonization resistance to
people with a current infection, we hypothesized that a fecal community
should be sufficient to restore colonization resistance to an uninfected
community. Therefore, we tested whether a fecal community transplant
(FCT) pre-treatment would prevent or clear \emph{C. difficile}
colonization and how variation in susceptibility to \emph{C. difficile}
infection would affect the effectiveness of FCT pre-treatment. After
testing the same FCT pre-treatment across different antibiotic-induced
susceptibilities, we sought to determine whether diluted FCT
pre-treatment could maintain the inhibition of \emph{C. difficile}
colonization and identify the bacterial populations associated with
colonization resistance and clearance.

\hypertarget{results}{%
\subsection{Results}\label{results}}

\textbf{Effect of fecal transplant on \emph{C. difficile} colonization
was not consistent across antibiotic treatments.} Our previous research
demonstrated that when mice were perturbed with different antibiotics,
there were antibiotic-specific changes to the microbial community that
resulted in different levels of colonization and clearance of
\emph{Clostridioides difficile} infection (14). Because each of these
treatments opened different niche spaces that \emph{C. difficile} could
fill, we hypothesized that the resulting community varied in the types
of bacteria required to recover colonization resistance. To test the
ability of the murine communities to recover colonization resistance, we
treated conventionally raised SPF C57BL/6 mice with either clindamycin,
cefoperazone, or streptomycin. After a short recovery period, the mice
were given either phosphate-saline buffer (PBS) or a fecal community
transplant via oral gavage (Figure 1). The fecal community was obtained
from untreated mice. One day after receiving the FCT, the mice were
challenged with 10\textsuperscript{3} \emph{C. difficile} 630 spores.
One day after the challenge, mice that were treated with either
clindamycin or cefoperazone and received the FCT pre-treatment had
similar amounts of \emph{C. difficile} colony forming units (CFU) as
those which received PBS. Among the clindamycin-treated mice, \emph{C.
difficile} colonization was cleared at similar rates regardless of
whether they received the FCT or PBS pre-treatments (Figure 2). For
cefoperazone-treated mice, \emph{C. difficile} colonized all of the
mice, but the mice that received the FCT pre-treatment cleared the
infection (Figure 2). For the streptomycin-treated mice, the FCT
pre-treatment resulted in either no detectable \emph{C. difficile}
colonization (8 of 14) or an infection that the community cleared within
5 days (Figure 2). For mice that would have normally had a persistent
infection, the FCT enabled them to clear the infection and in the
streptomycin-treated mice it was able to prevent infection entirely for
some mice.

\textbf{Diluted fecal communities prevented colonization and promoted
clearance for streptomycin-treated mice.} Next, we sought to test
whether mice that received a diluted FCT pre-treatment could still
benefit. To identify the minimally effective dilution, we repeated the
same experimental design (Figure 1) with the FCT diluted serially down
to 1:10\textsuperscript{5}. Since the FCT pre-treatment had no detected
effect in clindamycin-treated mice, we did not study those mice further.
Cefoperazone-treated mice pre-treated with diluted FCT, 1:10 and lower,
were not affected and were colonized throughout the experiment (Figure
3). Streptomycin-treated mice pre-treated with diluted FCT either
regained colonization resistance or were enabled to clear \emph{C.
difficile}. The streptomycin-treated mice pre-treated with FCT as dilute
as 1:10\textsuperscript{3} cleared \emph{C. difficile}. Some
streptomycin-treated mice pre-treated with FCT as dilute as
1:10\textsuperscript{2} had no \emph{C. difficile} CFU detected
throughout the length of the experiment. While more mice pre-treated
with lower FCT dilutions were colonized (FCT 6 of 14 were colonized,
1:10 10 of 12 were colonized, 1:10\textsuperscript{2} 10 of 14 were
colonized), the colonized mice that received the lower dilutions were
still able to clear \emph{C. difficile} (Figure S1). Thus, the reduced
fecal communities from the diluted FCT were able to restore colonization
resistance and promote clearance of \emph{C. difficile} in
streptomycin-treated mice.

The reduced fecal communities of the diluted FCT may have reduced
abundance and membership. We compared the FCT communities to determine
the differences between the dilutions. The most significant difference
between the communities of the FCT and its dilutions was the quantity of
the 16S rRNA gene, which decreased monotonically (Figure S2D). The FCT
dilutions of 1:10\textsuperscript{3} to 1:10\textsuperscript{5}) of the
had few samples with sufficient sequencing depth to provide bacterial
community information. The FCT and its dilutions were not significantly
different in either \(\alpha\)-diversity (number of operational
taxonomic units (OTUs) (S\textsubscript{obs}) or Inverse Simpson) or
\(\beta\)-diversity (\(\theta\)\textsubscript{YC}) (Figure S2A-C).
Populations of \emph{Acetatifactor}, \emph{Enterobacteriaceae},
\emph{Lactobacillus}, \emph{Ruminococcaceae}, and \emph{Turicibacter}
correlated with the FCT dilution factor (Figure S2E). Overall, the
abundance of the bacteria appeared to be largest difference between FCT
and its dilutions.

\textbf{Murine gut bacterial communities had not recovered their
diversity by the time of \emph{C. difficile} challenge.} To elucidate
the effects of the fecal community dilution on the murine gut bacterial
community and \emph{C. difficile} infection, we sequenced the V4 region
of the 16S rRNA gene from the fecal community. For the gut communities,
in comparison to the initial community (day -9), FCT pre-treatment did
not result in a significant recovery of diversity at the time of
\emph{C. difficile} challenge (day 0) for cefoperazone-treated mice
(Figure S3) or streptomycin-treated mice (Figure 4). At the end of the
experiment (day 10), the gut bacterial communities were more similar to
their initial community in \(\alpha\)-diversity (number of OTUs
(S\textsubscript{obs}) and Inverse Simpson diversity index) and
\(\beta\)-diversity (\(\theta\)\textsubscript{YC}) diversity. The mice
pre-treated with less dilute FCT were most similar to the initial
community structure, whereas, the mice pre-treated with more dilute FCT
resulted in little recovery of diversity, similar to the mice given PBS.
Thus, the less dilute FCT treatments did not result in restoration of
pre-antibiotic treatment community diversity at the time of \emph{C.
difficile} challenge but were sufficient to affect \emph{C. difficile}
colonization. This would suggest the effect was driven by the most
abundant populations.

\textbf{Gut bacterial community members are differentially abundant in
streptomycin-treated mice resistant to colonization.} Although there
were no significant differences in diversity at the time of challenge,
we next investigated how the individual bacterial populations were
different in the uncolonized streptomycin-treated mice pre-treated with
FCT. We used linear discriminant analysis (LDA) effect size (LEfSe)
analysis to identify OTUs within the fecal bacterial communities from
the streptomycin-treated mice that were differentially abundant between
uncolonized and colonized mice. The antibiotic treatment significantly
altered 99 OTUs (Figure S5), but on the day of \emph{C. difficile}
challenge only 7 OTUs were differentially abundant between colonized and
uncolonized communities (Figure 5A). Communities resistant to \emph{C.
difficile} colonization had more abundant populations of OTUs related to
\emph{Akkermansia}, \emph{Clostridiales}, \emph{Olsenella}, and
\emph{Porphyromonadaceae} and less abundant populations of an OTU
related to \emph{Enterobacteriaceae}. Thus, a small portion of OTUs,
relative to the changes due to streptomycin treatment, were
differentially abundant in mice that resisted \emph{C. difficile}
colonization compared to those that were colonized.

\textbf{Murine gut bacterial communities that cleared \emph{C.
difficile} colonization were more similar to the initial community.} To
better understand the differences in streptomycin-treated murine fecal
community that contributed to \emph{C. difficile} clearance, we compared
the communities that cleared \emph{C. difficile} to those that did not
at the time of challenge and 10 days post infection. Communities from
mice that cleared colonization were more similar to their initial
community at the end of the experiment than the mice that remained
colonized (Figure S4). At the time of \emph{C. difficile} challenge, 9
OTUs were differentially abundant between communities that remained
colonized to those that cleared colonization (Figure 5B). Communities
that cleared \emph{C. difficile} colonization had more abundant
populations of OTUs related to \emph{Porphyromonadaceae} and
\emph{Lachnospiraceae} and less abundant populations of OTUs related to
\emph{Acetatifactor}, \emph{Lachnospiraceae}, \emph{Olsenella},
\emph{Porphyromonadaceae}, and \emph{Salmonella}. At the end of the
experiment, 29 of the 34 differentially abundant OTUs were more abundant
in the mice that were able to clear the colonization (Figure 5C). The
relative abundance of OTUs related to \emph{Acetatifactor},
\emph{Anaeroplasma}, \emph{Enterococcus}, \emph{Lachnospiraceae},
\emph{Lactobacillus}, \emph{Porphyromonadaceae}, and
\emph{Ruminococcaceae} were higher in communities that cleared,
recovering in abundance from the streptomycin treatment. Multiple OTUs
related to \emph{Lachnospiraceae} and \emph{Porphyromonadaceae} (N = 14
and N = 5, respectively) were significant and accounted for greater
portions of the community (more than 10\%). However one
\emph{Porphyromonadaceae} population (OTU 5) and two
\emph{Lachnospiraceae} related populations (OTU 40 and 95) were more
abundant in the mice that remain colonized. Thus, as more of the gut
bacterial members returned to their initial abundance, there was a
greater likelihood of clearing \emph{C. difficile}.

\textbf{Negative associations dominated the interactions between the gut
bacterial community and \emph{C. difficile} in streptomycin-treated
mice.} In streptomycin-treated mice, pre-treatment with FCT and its
dilutions had different effects on the bacterial community members which
resulted in different community relative abundance and \emph{C.
difficile} colonization dynamics. We quantified the relationships
occurring throughout this experiment using SPIEC-EASI (sparse inverse
covariance estimation for ecological association inference) to construct
a conditional independence network. Here, we focused on the associations
of the \emph{C. difficile} subnetwork (Figure 6). \emph{C. difficile}
CFU had positive associations with populations of OTUs related to
\emph{Enterobacteriaceae} (OTU 4) and \emph{Peptostreptococcaceae} (OTU
19). OTUs related to \emph{Clostridiales} (OTU 27),
\emph{Lachnospiraceae} (OTUs 15, 51, and 83), and
\emph{Porphyromonadaceae} (OTUs 23, 25, and 29) had negative
associations with \emph{C. difficile}, as well as the OTUs related to
\emph{Enterobacteriaceae}, and \emph{Peptostreptococcaceae}. Overall,
the majority of the associations between \emph{C. difficile} and the gut
bacterial community in streptomycin-treated mice were negative. This
suggests this subset of the community may be driving the inhibition of
\emph{C. difficile} in streptomycin-treated communities.

\hypertarget{discussion}{%
\subsection{Discussion}\label{discussion}}

Transplanting the fecal community from untreated mice to
antibiotic-treated mice prior to being challenged with \emph{C.
difficile} varied in effectiveness based on the antibiotic treatment.
This indicated that FCT pre-treatment can prevent \emph{C. difficile}
colonization in an antibiotic-specific manner. Additionally, by diluting
the FCT we were able to narrow the community changes responsible for the
effect to the most abundant OTUs. Overall, these results show that a
reduced fecal community can assist a perturbed microbiota in preventing
or resisting \emph{C. difficile} colonization but the effect was
dependent on the antibiotic that was given.

By diluting the FCT we were able to narrow the definition of the minimal
community features that restored colonization resistance. Bacterial
interactions with \emph{C. difficile} were associated with the identity,
abundance, and functions of adjacent bacteria. Ghimire \emph{et al.}
recently showed individual species that inhibited \emph{C. difficile} in
co-culture but when other inhibitory species were added the overall
effect on \emph{C. difficile} was changed, in some cases to increase
\emph{C. difficile} growth (15). Based on these observations from their
bottom-up approach, it is unclear how more complex combinations would
affect the inhibition of \emph{C. difficile}. So instead, we sought to
find an inhibitory community using a top-down approach and begin with an
inhibitory community. In a recent top-down approach, Auchtung \emph{et
al.} recently developed a set of reduced communities from human fecal
communities that were grown in minibioreactor arrays and tested for
inhibition first \emph{in vitro} then in a mouse model (16). They found
four reduced communities that were able to reduce \emph{C. difficile}
colonization but with varied effect in a mouse model with the same gut
microbiota. One way they reduced the community was through diluting the
initial fecal sample. In our experiments, we began with a fresh whole
fecal community to first determine if inhibition was possible. In the
conditions which \emph{C. difficile} was inhibited, with cefoperazone
and streptomycin, we diluted the FCT to determine the minimal community
which maintained inhibition. Cefoperazone-treated mice were unable to
maintain inhibition of \emph{C. difficile} with diluted FCT
pre-treatments and \emph{C. difficile} remained colonized.
Streptomycin-treated mice were able to maintain inhibition with diluted
FCT pre-treatment. While the diluted FCTs had similar diversity and
bacterial abundances, the differences in effect on \emph{C. difficile}
revealed the minimal changes associated with either colonization
resistance or clearance.

We previously hypothesized that mice treated with either clindamycin,
cefoperazone, or streptomycin would not have the same bacterial
community changes associated with \emph{C. difficile} clearance (14). In
that set of experiments, the dose of the antibiotic was varied to
titrate changes to the community and determine what changes allow
\emph{C. difficile} to colonize and then be spontaneously cleared. We
observed antibiotic-specific changes associated with \emph{C. difficile}
clearance. The data presented here complement those observations. For
clindamycin-treated mice, there was no difference in colonization,
clearance or relative abundance between PBS and FCT pre-treatment.
\emph{C. difficle} had similar colonization dynamics. It is possible
that there was insufficient time for the FCT to engraft. However, when
we added more time between clindamycin treatment and \emph{C. difficile}
challenge, \emph{C. difficile} was unable to colonize (data not shown).
Therefore, clindamycin-treated mice appeared to have been naturally
recovering inhibition to \emph{C. difficile}, which was unaffected by
the FCT pre-treatment. For cefoperazone-treated mice, the FCT
pre-treatment enabled the gut microbiota to eliminate the colonization
but only in its most concentrated dose. This observation supports our
previous discussion (14), indicating that the cefoperazone-treated
community is more sensitive to the amount of FCT it receives since
cefoperazone reduced many bacterial groups and associations (Figure S6).
As we previously hypothesized, streptomycin-treated mice were enabled to
clear with a subset of the community, with the FCT pre-treatment diluted
1:10\textsuperscript{3}. Since we titrated the FCT dilutions, we could
compare the bacterial communities of the mice which gained the ability
to clear \emph{C. difficile} to the mice that received the next dilution
which could not to elucidate the minimal relative abundance differences.
In agreement with previous studies, OTUs related to
\emph{Lachnospiraceae}, \emph{Porphyromonadaceae}, and
\emph{Ruminococcaceae} increased with the clearance of \emph{C.
difficle} in the streptomycin-treated mice (14, 17--21). These data
agree with our previous hypothesis that a reduced fecal community would
only be able to promote clearance of \emph{C. difficile} in
streptomycin-treated mice.

In addition to clearing \emph{C. difficile}, a reduced fecal community
restored colonization resistance to streptomycin-treated mice. Mice that
received FCT pre-treatment as dilute as 1:10\textsuperscript{2} were not
colonized to a detectable level. While restoring colonization resistance
is not novel (22), here we have shown that the restoration of
colonization resistance is dependent on the community perturbation and
the fecal community being transplanted. As we identified community
members associated with clearance, OTUs related to \emph{Akkermansia},
\emph{Olsenella}, and \emph{Porphyromonadaceae} were more abundant and
an OTU related to \emph{Enterobacteriaceae} was less abundant at the
time of \emph{C. difficile} challenge. \emph{Enterobacteriaceae} has
been associated with \emph{C. difficile} colonization and inflammation
(14, 23, 24). Larger populations of \emph{Akkermansia} were associated
with preventing colonization, which we had previously observed,
potentially indicating the maintenance of a protective mucus layer (14,
25--27). Increased populations of a select set of OTUs related to
\emph{Porphyromonadaceae} were also more abundant in mice that were
resistant to colonization. \emph{Porphyromonadaceae} may be inhibiting
\emph{C. difficile} via buytrate and acetate production, which has been
associated with successful FMT treatments (28--30). Different
populations of OTUs associated with \emph{Porphyromonadaceae} were
associated with colonization resistance than with colonization
clearance. These colonization resistance-associated OTUs (OTUs 8, 25 and
29) may have OTU-specific functions or dependent abundances of members
of the community, such as \emph{Akkermansia} and
\emph{Enterobacteriaceae}. With our top-down approach, we reduced the
number of gut bacterial community members that were associated with
colonization resistance in streptomycin-treated mice.

Further investigation into the heterogeneity of CDI will help to
elucidate the niche range \emph{C. difficile} and the interventions to
eliminate them. Here we were limited by our experimental design and
methods to refining our understanding of colonization resistance
restoration in streptomycin-treated mice. Future studies can expand
beyond the presence and abundance of the bacterial groups and
investigate the metabolites and host immune response. A refined
understanding of the bacteria, metabolites and host response can help
develop more targeted therapies to restore \emph{C. difficile}
colonization resistance. Additionally, building up experimental design
to incorporate more FCT treatment variations or inoculation regimens
could expand our understanding of the necessary components for
colonization resistance for each antibiotic treatment. We designed our
experiments to closely match previous mouse models for CDI and added
days prior to \emph{C. difficile} challenge for the FCT treatment (14,
31, 32). It may be possible to restore colonization resistance to
clindamycin or cefoperazone if the antibiotic treatment, recovery
period, and FCT treatment were modified to allow the FCT to have an
effect. Other methods could be used to make the mice susceptible to CDI
an then tested for the effectiveness of the FCT treatment (18, 33).
Further modification and characterization of the fecal communities could
reduce the necessary community members and metabolites to promote
colonization resistance. The results from these additional studies could
expand upon our limitations and reveal specific bacterial communities
that could restore \emph{C. difficile} colonization resistance for each
susceptibility.

We have demonstrated that a reduced bacterial community can restore
colonization resistance but the effect of the community and the bacteria
that colonized was dependent on the specific changes to the community
that were caused by each antibiotic. When transplanting the fecal
community into antibiotic-induced susceptible mice, only mice treated
with streptomycin were able to restore colonization resistance. Previous
studies that have identified reduced communities in a murine model using
a homogeneous gut microbiota with a bottom-up approach (7, 19).
Treatments supplementing the gut microbiota would benefit from being
tested in different communities susceptible to CDI. Further research is
necessary to characterize the specific niche spaces \emph{C. difficile}
of susceptibilities communities and the specific requirements fill those
spaces. Then it may be possible to identify people with gut microbiota
that are susceptible to CDI and develop targeted reduced bacterial
communities to recover colonization resistance and reduce the risk of
CDI.

\hypertarget{materials-and-methods}{%
\subsection{Materials and Methods}\label{materials-and-methods}}

\textbf{Animal care.} Mice used in experiments were 6- to 13-week old
conventionally reared SPF male C57BL/6 mice obtained from a single
breeding colony at the University of Michigan. During the experiment,
mice were housed with two or three mice per cage. All murine experiments
were approved by the University of Michigan Animal Care and Use
Committee (IACUC) under protocol number PRO00006983.

\textbf{Antibiotic administration.} Antibiotics were chosen and
administered based on previous studies (14, 31, 32). Cefoperazone,
clindamycin, and streptomycin treatment produced diverse communities and
responses to CDI. Mice were given either cefoperazone, clindamycin, or
streptomycin. Cefoperazone (0.5 mg/ml) and streptomycin (5 mg/ml) were
administered via drinking water \emph{ad libitum} for 5 days, beginning
9 days prior to \emph{C. difficile} challenge. Antibiotic water was
replaced every two days. Clindamycin (10 mg/kg) was injected into the
intraperitoneal space, 2 days prior to challenge with \emph{C.
difficile}. All antibiotics were filter sterilized with a 0.22 \(\mu\)m
syringe filter prior to use.

\textbf{Fecal community transplants.} Fecal pellets were collected from
similar aged C57BL/6 mice not being used in an experiment the day of the
fecal community transplants. 15-20 pellets were collected and weighed.
The fecal pellets were homogenized weight per weight in phosphate-saline
buffer (PBS) containing 15\% glycerol (fecal community transplant, FCT)
in anaerobic conditions. The FCT was serially diluted in PBS containing
15\% glycerol down to 1:10\textsuperscript{5} fecal dilution and
aliquoted into tubes for gavaging into mice. One set of aliquots were
frozen at -80\(^\circ\)C to be used the following day for the
cefoperazone and streptomycin experiments. Frozen aliquots were thawed
at 37\(^\circ\)C for 5 minutes prior to being used. All fecal community
dilutions were centrifuged at 7500 RPM for 60 seconds and the
supernatant was used for inoculation. Mice were inoculated with 100 uL
of the fecal dilution oral via a 21 gauge gavage needle. Fecal community
transplants were administered from most dilute to least, which began
with mice receiving PBS and finished with mice receiving FCT. Aliquots
were frozen at -80\(^\circ\)C after use for sequencing. These
experiments were repeated 8 times with a different starting source each
time. This method was adapted from our previous study (18).

\textbf{16S rRNA quantitative real-time PCR.} Quantitative analysis of
16S rRNA in fecal community dilutions used for FCT was carried out using
quantitative real-time PCR using primers and cycler conditions specified
previously (34). Reaction volumes were prepared using 6 uL of SYBRTM
Green PCR Master Mix (Applied Biosciences Ref 4344463), 1 uL each
forward and reverse primer, and 2 uL sample DNA template. All qPCR
reactions were run on a LightCycler\textregistered ~96 (Roche Ref
05815916001) using instrument-specific plates and seals.

\textbf{\emph{C. difficile} challenge.} For experiments using
streptomycin or cefoperazone, mice were given untreated drinking water
for 96 hours before challenging with \emph{C. difficile} strain
630\(\Delta\)erm spores. For experiments using clindamycin, mice were
given untreated drinking water for 48 hours the time of the
intraperitoneal injection and being challenged with \emph{C. difficile}
strain 630\(\Delta\)erm spores. This time frame was designed to closely
replicate the previous mouse model (14, 31, 32), with the insertion of a
day (clindamycin) or two (cefoperazone and streptomycin) for inoculating
the mice with the fecal communities. \emph{C. difficile} spores were
aliquoted from a single spore stock stored at 4\(^\circ\)C. Spore
concentration was determined two days prior to the day of challenge
(35). Mice were inoculated with 10\textsuperscript{3} \emph{C.
difficile} spores via oral gavage. After inoculating the mice, remaining
spore solution was serially diluted and plated to confirm the spore
concentration.

\textbf{Sample collection.} Fecal samples were collected prior to
administering antibiotics, after antibiotics were removed, prior to
\emph{C. difficile} challenge and on each of the 10 days post infection.
Approximately 15 mg of each fecal sample was collected and weighed for
plating \emph{C. difficile} colony forming units (CFU) and the remaining
sample was frozen at -80\(^\circ\)C for later sequencing. The weighed
fecal samples were anaerobically serially diluted in PBS, plated on
TCCFA plates, and incubated at 37\(^\circ\)C for 24 hours. The resultant
colonies were enumerated to determine the \emph{C. difficile} CFUs (36).

\textbf{DNA sequencing.} Total bacterial DNA was extracted from the
frozen samples by the MOBIO PowerSoil-htp 96-well soil DNA isolation
kit. We amplified the 16S rRNA gene V4 region and the amplicons were
sequenced on an Illumina MiSeq as described previously (37).

\textbf{Sequence curation.} Sequences were processed using mothur
(v.1.44.1) (37, 38). We used a 3\% dissimilarity cutoff to group
sequences into operational taxonomic units (OTUs) and a naive Bayesian
classifier with the Ribosomal Database Project training set (version 16)
to assign taxonomic classifications to OTUs (39). We sequenced a mock
community of a known 16S rRNA gene sequences and composition. We
processed this mock community in parallel with our samples to determine
the error rate for our sequence curation, which resulted in an error
rate of 0.029\%.

\textbf{Statistical analysis and modeling.} We calculated diversity
metrics in mothur. For \(\alpha\)-diversity comparisons, we calculated
the number of OTUs (S\textsubscript{obs}) and the Inverse Simpson
diversity index. For \(\beta\)-diversity comparisons, we calculated
dissimilarity matrices based on metric of Yue and Clayton
(\(\theta\)\textsubscript{YC}) (40). We averaged 1000 sub-samples of
2,480 counts per sample, or rarified, to limit uneven sampling biases.
We tested for differences in relative abundance between outcomes with
LEfSe in mothur (41). All other statistical analyses and data
visualization was completed in R (v4.0.5) with the tidyverse package
(v1.3.1). Pairwise comparisons of \(\alpha\)-diversity
(S\textsubscript{obs} and Inverse Simpson), \(\beta\)-diversity
(\(\theta\)\textsubscript{YC}), were calculated by pairwise Wilcoxon
rank sum test. Correlations between bacterial genera and fecal community
dilution were calculated using the Spearman correlation. \emph{P} values
were corrected for multiple comparisons with a Benjamini and Hochberg
adjustment for a type I error rate of 0.05 (42). For streptomycin
experiments, conditional independence networks were calculated from the
day 1 through 5 samples of all mice using SPIEC-EASI (sparse inverse
covariance estimation for ecological association inference) methods from
the SpiecEasi R package after optimizing lambda to 0.001 with a network
stability between 0.045 and 0.05 (v1.0.7) (43).

\textbf{Code availability.} Scripts necessary to reproduce our analysis
and this paper are available in an online repository
(\url{https://github.com/SchlossLab/Lesniak_restoreCR_mBio_2022}).

\textbf{Sequence data accession number.} All 16S rRNA gene sequence data
and associated metadata are available through the Sequence Read Archive
via accession SRP373949.

\hypertarget{acknowledgements}{%
\subsection{Acknowledgements}\label{acknowledgements}}

This work was supported by several grants from the National Institutes
for Health R01GM099514, U19AI090871, U01AI12455, and P30DK034933.
Additionally, NAL was supported by the Molecular Mechanisms of Microbial
Pathogenesis training grant (NIH T32 AI007528). The funding agencies had
no role in study design, data collection and analysis, decision to
publish, or preparation of the manuscript.

\hypertarget{author-contributions}{%
\subsection{Author contributions}\label{author-contributions}}

Conceptualization: N.A.L., S.T., P.D.S.; Data curation: N.A.L., L.B.,
K.M.; Formal analysis: N.A.L., ; Investigation: N.A.L., S.T., A.H.,
A.T., J.C., L.B., K.M., P.D.S.; Methodology: N.A.L., S.T., P.D.S.;
Resources: N.A.L., S.T., L.B., K.M., P.D.S.; Software: N.A.L.;
Visualization: N.A.L., P.D.S.; Writing - original draft: NAL; Writing -
review \& editing: N.A.L., S.T., A.H., A.T., J.C., L.B., K.B., P.D.S.;
Funding acquisition: P.D.S.; Project administration: P.D.S.;
Supervision: P.D.S.

\newpage

\hypertarget{references}{%
\subsection{References}\label{references}}

\hypertarget{refs}{}
\begin{cslreferences}
\leavevmode\hypertarget{ref-Ducarmon2019}{}%
1. \textbf{Ducarmon QR}, \textbf{Zwittink RD}, \textbf{Hornung BVH},
\textbf{Schaik W van}, \textbf{Young VB}, \textbf{Kuijper EJ}. 2019. Gut
microbiota and colonization resistance against bacterial enteric
infection. Microbiology and Molecular Biology Reviews \textbf{83}.
doi:\href{https://doi.org/10.1128/mmbr.00007-19}{10.1128/mmbr.00007-19}.

\leavevmode\hypertarget{ref-Vardakas2016}{}%
2. \textbf{Vardakas KZ}, \textbf{Trigkidis KK}, \textbf{Boukouvala E},
\textbf{Falagas ME}. 2016. \emph{Clostridium difficile} infection
following systemic antibiotic administration in randomised controlled
trials: A systematic review and meta-analysis. International Journal of
Antimicrobial Agents \textbf{48}:1--10.
doi:\href{https://doi.org/10.1016/j.ijantimicag.2016.03.008}{10.1016/j.ijantimicag.2016.03.008}.

\leavevmode\hypertarget{ref-Cammarota2014}{}%
3. \textbf{Cammarota G}, \textbf{Ianiro G}, \textbf{Gasbarrini A}. 2014.
Fecal microbiota transplantation for the treatment of \emph{Clostridium
difficile} infection. Journal of Clinical Gastroenterology
\textbf{48}:693--702.
doi:\href{https://doi.org/10.1097/mcg.0000000000000046}{10.1097/mcg.0000000000000046}.

\leavevmode\hypertarget{ref-Beran2022}{}%
4. \textbf{Beran A}, \textbf{Sharma S}, \textbf{Ghazaleh S},
\textbf{Lee-Smith W}, \textbf{Aziz M}, \textbf{Kamal F}, \textbf{Acharya
A}, \textbf{Adler DG}. 2022. Predictors of fecal microbiota transplant
failure in \emph{Clostridioides difficile} infection. Journal of
Clinical Gastroenterology \textbf{Publish Ahead of Print}.
doi:\href{https://doi.org/10.1097/mcg.0000000000001667}{10.1097/mcg.0000000000001667}.

\leavevmode\hypertarget{ref-DeFilipp2019}{}%
5. \textbf{DeFilipp Z}, \textbf{Bloom PP}, \textbf{Soto MT},
\textbf{Mansour MK}, \textbf{Sater MRA}, \textbf{Huntley MH},
\textbf{Turbett S}, \textbf{Chung RT}, \textbf{Chen Y-B},
\textbf{Hohmann EL}. 2019. Drug-resistant e. Coli bacteremia transmitted
by fecal microbiota transplant. New England Journal of Medicine
\textbf{381}:2043--2050.
doi:\href{https://doi.org/10.1056/nejmoa1910437}{10.1056/nejmoa1910437}.

\leavevmode\hypertarget{ref-Tvede1989}{}%
6. \textbf{Tvede M}, \textbf{Rask-Madsen J}. 1989. Bacteriotherapy for
chronic relapsing \emph{Clostridium difficile} diarrhoea in six
patients. The Lancet \textbf{333}:1156--1160.
doi:\href{https://doi.org/10.1016/s0140-6736(89)92749-9}{10.1016/s0140-6736(89)92749-9}.

\leavevmode\hypertarget{ref-Lawley2012}{}%
7. \textbf{Lawley TD}, \textbf{Clare S}, \textbf{Walker AW},
\textbf{Stares MD}, \textbf{Connor TR}, \textbf{Raisen C},
\textbf{Goulding D}, \textbf{Rad R}, \textbf{Schreiber F},
\textbf{Brandt C}, \textbf{Deakin LJ}, \textbf{Pickard DJ},
\textbf{Duncan SH}, \textbf{Flint HJ}, \textbf{Clark TG},
\textbf{Parkhill J}, \textbf{Dougan G}. 2012. Targeted restoration of
the intestinal microbiota with a simple, defined bacteriotherapy
resolves relapsing \emph{Clostridium difficile} disease in mice. PLoS
Pathogens \textbf{8}:e1002995.
doi:\href{https://doi.org/10.1371/journal.ppat.1002995}{10.1371/journal.ppat.1002995}.

\leavevmode\hypertarget{ref-Petrof2013}{}%
8. \textbf{Petrof EO}, \textbf{Gloor GB}, \textbf{Vanner SJ},
\textbf{Weese SJ}, \textbf{Carter D}, \textbf{Daigneault MC},
\textbf{Brown EM}, \textbf{Schroeter K}, \textbf{Allen-Vercoe E}. 2013.
Stool substitute transplant therapy for the eradication of
\emph{Clostridium difficile} infection: ``RePOOPulating'' the gut.
Microbiome \textbf{1}.
doi:\href{https://doi.org/10.1186/2049-2618-1-3}{10.1186/2049-2618-1-3}.

\leavevmode\hypertarget{ref-Kao2021}{}%
9. \textbf{Kao D}, \textbf{Wong K}, \textbf{Franz R}, \textbf{Cochrane
K}, \textbf{Sherriff K}, \textbf{Chui L}, \textbf{Lloyd C},
\textbf{Roach B}, \textbf{Bai AD}, \textbf{Petrof EO},
\textbf{Allen-Vercoe E}. 2021. The effect of a microbial ecosystem
therapeutic (MET-2) on recurrent \emph{Clostridioides difficile}
infection: A phase 1, open-label, single-group trial. The Lancet
Gastroenterology \& Hepatology \textbf{6}:282--291.
doi:\href{https://doi.org/10.1016/s2468-1253(21)00007-8}{10.1016/s2468-1253(21)00007-8}.

\leavevmode\hypertarget{ref-Khanna2016}{}%
10. \textbf{Khanna S}, \textbf{Pardi DS}, \textbf{Kelly CR},
\textbf{Kraft CS}, \textbf{Dhere T}, \textbf{Henn MR}, \textbf{Lombardo
M-J}, \textbf{Vulic M}, \textbf{Ohsumi T}, \textbf{Winkler J},
\textbf{Pindar C}, \textbf{McGovern BH}, \textbf{Pomerantz RJ},
\textbf{Aunins JG}, \textbf{Cook DN}, \textbf{Hohmann EL}. 2016. A novel
microbiome therapeutic increases gut microbial diversity and prevents
recurrent \emph{Clostridium difficile} infection. Journal of Infectious
Diseases \textbf{214}:173--181.
doi:\href{https://doi.org/10.1093/infdis/jiv766}{10.1093/infdis/jiv766}.

\leavevmode\hypertarget{ref-McGovern2020}{}%
11. \textbf{McGovern BH}, \textbf{Ford CB}, \textbf{Henn MR},
\textbf{Pardi DS}, \textbf{Khanna S}, \textbf{Hohmann EL},
\textbf{O'Brien EJ}, \textbf{Desjardins CA}, \textbf{Bernardo P},
\textbf{Wortman JR}, \textbf{Lombardo M-J}, \textbf{Litcofsky KD},
\textbf{Winkler JA}, \textbf{McChalicher CWJ}, \textbf{Li SS},
\textbf{Tomlinson AD}, \textbf{Nandakumar M}, \textbf{Cook DN},
\textbf{Pomerantz RJ}, \textbf{Aunin≈° JG}, \textbf{Trucksis M}. 2020.
SER-109, an investigational microbiome drug to reduce recurrence after
\emph{Clostridioides difficile} infection: Lessons learned from a phase
2 trial. Clinical Infectious Diseases \textbf{72}:2132--2140.
doi:\href{https://doi.org/10.1093/cid/ciaa387}{10.1093/cid/ciaa387}.

\leavevmode\hypertarget{ref-Taur2018}{}%
12. \textbf{Taur Y}, \textbf{Coyte K}, \textbf{Schluter J},
\textbf{Robilotti E}, \textbf{Figueroa C}, \textbf{Gjonbalaj M},
\textbf{Littmann ER}, \textbf{Ling L}, \textbf{Miller L},
\textbf{Gyaltshen Y}, \textbf{Fontana E}, \textbf{Morjaria S},
\textbf{Gyurkocza B}, \textbf{Perales M-A}, \textbf{Castro-Malaspina H},
\textbf{Tamari R}, \textbf{Ponce D}, \textbf{Koehne G}, \textbf{Barker
J}, \textbf{Jakubowski A}, \textbf{Papadopoulos E}, \textbf{Dahi P},
\textbf{Sauter C}, \textbf{Shaffer B}, \textbf{Young JW}, \textbf{Peled
J}, \textbf{Meagher RC}, \textbf{Jenq RR}, \textbf{Brink MRM van den},
\textbf{Giralt SA}, \textbf{Pamer EG}, \textbf{Xavier JB}. 2018.
Reconstitution of the gut microbiota of antibiotic-treated patients by
autologous fecal microbiota transplant. Science Translational Medicine
\textbf{10}.
doi:\href{https://doi.org/10.1126/scitranslmed.aap9489}{10.1126/scitranslmed.aap9489}.

\leavevmode\hypertarget{ref-Reigadas2021}{}%
13. \textbf{Reigadas E}, \textbf{Prehn J van}, \textbf{Falcone M},
\textbf{Fitzpatrick F}, \textbf{Vehreschild MJGT}, \textbf{Kuijper EJ},
\textbf{Bouza E}. 2021. How to: Prophylactic interventions for
prevention of \emph{Clostridioides difficile} infection. Clinical
Microbiology and Infection \textbf{27}:1777--1783.
doi:\href{https://doi.org/10.1016/j.cmi.2021.06.037}{10.1016/j.cmi.2021.06.037}.

\leavevmode\hypertarget{ref-Lesniak2021}{}%
14. \textbf{Lesniak NA}, \textbf{Schubert AM}, \textbf{Sinani H},
\textbf{Schloss PD}. 2021. Clearance of \emph{Clostridioides difficile}
colonization is associated with antibiotic-specific bacterial changes.
mSphere \textbf{6}.
doi:\href{https://doi.org/10.1128/msphere.01238-20}{10.1128/msphere.01238-20}.

\leavevmode\hypertarget{ref-Ghimire2019}{}%
15. \textbf{Ghimire S}, \textbf{Roy C}, \textbf{Wongkuna S},
\textbf{Antony L}, \textbf{Maji A}, \textbf{Keena MC}, \textbf{Foley A},
\textbf{Scaria J}. 2020. Identification of \emph{Clostridioides
difficile}-inhibiting gut commensals using culturomics, phenotyping, and
combinatorial community assembly. mSystems \textbf{5}.
doi:\href{https://doi.org/10.1128/msystems.00620-19}{10.1128/msystems.00620-19}.

\leavevmode\hypertarget{ref-Auchtung2020}{}%
16. \textbf{Auchtung JM}, \textbf{Preisner EC}, \textbf{Collins J},
\textbf{Lerma AI}, \textbf{Britton RA}. 2020. Identification of
simplified microbial communities that inhibit \emph{Clostridioides
difficile} infection through dilution/extinction. mSphere \textbf{5}.
doi:\href{https://doi.org/10.1128/msphere.00387-20}{10.1128/msphere.00387-20}.

\leavevmode\hypertarget{ref-Tomkovich2020}{}%
17. \textbf{Tomkovich S}, \textbf{Stough JMA}, \textbf{Bishop L},
\textbf{Schloss PD}. 2020. The initial gut microbiota and response to
antibiotic perturbation influence \emph{Clostridioides difficile}
clearance in mice. mSphere \textbf{5}.
doi:\href{https://doi.org/10.1128/msphere.00869-20}{10.1128/msphere.00869-20}.

\leavevmode\hypertarget{ref-Tomkovich2021}{}%
18. \textbf{Tomkovich S}, \textbf{Taylor A}, \textbf{King J},
\textbf{Colovas J}, \textbf{Bishop L}, \textbf{McBride K},
\textbf{Royzenblat S}, \textbf{Lesniak NA}, \textbf{Bergin IL},
\textbf{Schloss PD}. 2021. An osmotic laxative renders mice susceptible
to prolonged \emph{Clostridioides difficile} colonization and hinders
clearance. mSphere \textbf{6}.
doi:\href{https://doi.org/10.1128/msphere.00629-21}{10.1128/msphere.00629-21}.

\leavevmode\hypertarget{ref-Buffie2015}{}%
19. \textbf{Buffie CG}, \textbf{Bucci V}, \textbf{Stein RR},
\textbf{McKenney PT}, \textbf{Ling L}, \textbf{Gobourne A}, \textbf{No
D}, \textbf{Liu H}, \textbf{Kinnebrew M}, \textbf{Viale A},
\textbf{Littmann E}, \textbf{Brink MRM van den}, \textbf{Jenq RR},
\textbf{Taur Y}, \textbf{Sander C}, \textbf{Cross JR}, \textbf{Toussaint
NC}, \textbf{Xavier JB}, \textbf{Pamer EG}. 2014. Precision microbiome
reconstitution restores bile acid mediated resistance to
\emph{Clostridium difficile}. Nature \textbf{517}:205--208.
doi:\href{https://doi.org/10.1038/nature13828}{10.1038/nature13828}.

\leavevmode\hypertarget{ref-Reeves2012}{}%
20. \textbf{Reeves AE}, \textbf{Koenigsknecht MJ}, \textbf{Bergin IL},
\textbf{Young VB}. 2012. Suppression of \emph{Clostridium difficile} in
the gastrointestinal tracts of germfree mice inoculated with a murine
isolate from the family Lachnospiraceae. Infection and Immunity
\textbf{80}:3786--3794.
doi:\href{https://doi.org/10.1128/iai.00647-12}{10.1128/iai.00647-12}.

\leavevmode\hypertarget{ref-Leslie2019}{}%
21. \textbf{Leslie JL}, \textbf{Vendrov KC}, \textbf{Jenior ML},
\textbf{Young VB}. 2019. The gut microbiota is associated with clearance
of \emph{Clostridium difficile} infection independent of adaptive
immunity. mSphere \textbf{4}.
doi:\href{https://doi.org/10.1128/mspheredirect.00698-18}{10.1128/mspheredirect.00698-18}.

\leavevmode\hypertarget{ref-NagaoKitamoto2020}{}%
22. \textbf{Nagao-Kitamoto H}, \textbf{Leslie JL}, \textbf{Kitamoto S},
\textbf{Jin C}, \textbf{Thomsson KA}, \textbf{Gillilland MG},
\textbf{Kuffa P}, \textbf{Goto Y}, \textbf{Jenq RR}, \textbf{Ishii C},
\textbf{Hirayama A}, \textbf{Seekatz AM}, \textbf{Martens EC},
\textbf{Eaton KA}, \textbf{Kao JY}, \textbf{Fukuda S}, \textbf{Higgins
PDR}, \textbf{Karlsson NG}, \textbf{Young VB}, \textbf{Kamada N}. 2020.
Interleukin-22-mediated host glycosylation prevents \emph{Clostridioides
difficile} infection by modulating the metabolic activity of the gut
microbiota. Nature Medicine \textbf{26}:608--617.
doi:\href{https://doi.org/10.1038/s41591-020-0764-0}{10.1038/s41591-020-0764-0}.

\leavevmode\hypertarget{ref-Byndloss2017}{}%
23. \textbf{Byndloss MX}, \textbf{Olsan EE}, \textbf{Rivera-Ch√°vez F},
\textbf{Tiffany CR}, \textbf{Cevallos SA}, \textbf{Lokken KL},
\textbf{Torres TP}, \textbf{Byndloss AJ}, \textbf{Faber F}, \textbf{Gao
Y}, \textbf{Litvak Y}, \textbf{Lopez CA}, \textbf{Xu G}, \textbf{Napoli
E}, \textbf{Giulivi C}, \textbf{Tsolis RM}, \textbf{Revzin A},
\textbf{Lebrilla CB}, \textbf{B√§umler AJ}. 2017. Microbiota-activated
PPAR- signaling inhibits dysbiotic Enterobacteriaceae expansion. Science
\textbf{357}:570--575.
doi:\href{https://doi.org/10.1126/science.aam9949}{10.1126/science.aam9949}.

\leavevmode\hypertarget{ref-Winter2013}{}%
24. \textbf{Winter SE}, \textbf{Lopez CA}, \textbf{B√§umler AJ}. 2013.
The dynamics of gut-associated microbial communities during
inflammation. EMBO reports \textbf{14}:319--327.
doi:\href{https://doi.org/10.1038/embor.2013.27}{10.1038/embor.2013.27}.

\leavevmode\hypertarget{ref-Lesniak2022}{}%
25. \textbf{Lesniak NA}, \textbf{Schubert AM}, \textbf{Flynn KJ},
\textbf{Leslie JL}, \textbf{Sinani H}, \textbf{Bergin IL}, \textbf{Young
VB}, \textbf{Schloss PD}. 2022. The gut bacterial community potentiates
\emph{Clostridioides difficile} infection severity.
doi:\href{https://doi.org/10.1101/2022.01.31.478599}{10.1101/2022.01.31.478599}.

\leavevmode\hypertarget{ref-Nakashima2021}{}%
26. \textbf{Nakashima T}, \textbf{Fujii K}, \textbf{Seki T},
\textbf{Aoyama M}, \textbf{Azuma A}, \textbf{Kawasome H}. 2021. Novel
gut microbiota modulator, which markedly increases \emph{Akkermansia
muciniphila} occupancy, ameliorates experimental colitis in rats.
Digestive Diseases and Sciences.
doi:\href{https://doi.org/10.1007/s10620-021-07131-x}{10.1007/s10620-021-07131-x}.

\leavevmode\hypertarget{ref-Stein2013}{}%
27. \textbf{Stein RR}, \textbf{Bucci V}, \textbf{Toussaint NC},
\textbf{Buffie CG}, \textbf{R√§tsch G}, \textbf{Pamer EG}, \textbf{Sander
C}, \textbf{Xavier JB}. 2013. Ecological modeling from time-series
inference: Insight into dynamics and stability of intestinal microbiota.
PLoS Computational Biology \textbf{9}:e1003388.
doi:\href{https://doi.org/10.1371/journal.pcbi.1003388}{10.1371/journal.pcbi.1003388}.

\leavevmode\hypertarget{ref-Flynn2018}{}%
28. \textbf{Flynn KJ}, \textbf{Ruffin MT}, \textbf{Turgeon DK},
\textbf{Schloss PD}. 2018. Spatial variation of the native colon
microbiota in healthy adults. Cancer Prevention Research
\textbf{11}:393--402.
doi:\href{https://doi.org/10.1158/1940-6207.capr-17-0370}{10.1158/1940-6207.capr-17-0370}.

\leavevmode\hypertarget{ref-Guilloux2021}{}%
29. \textbf{Guilloux C-A}, \textbf{Lamoureux C}, \textbf{Beauruelle C},
\textbf{H√©ry-Arnaud G}. 2021. Porphyromonas: A neglected potential key
genus in human microbiomes. Anaerobe \textbf{68}:102230.
doi:\href{https://doi.org/10.1016/j.anaerobe.2020.102230}{10.1016/j.anaerobe.2020.102230}.

\leavevmode\hypertarget{ref-Seekatz2018}{}%
30. \textbf{Seekatz AM}, \textbf{Theriot CM}, \textbf{Rao K},
\textbf{Chang Y-M}, \textbf{Freeman AE}, \textbf{Kao JY}, \textbf{Young
VB}. 2018. Restoration of short chain fatty acid and bile acid
metabolism following fecal microbiota transplantation in patients with
recurrent \emph{Clostridium difficile} infection. Anaerobe
\textbf{53}:64--73.
doi:\href{https://doi.org/10.1016/j.anaerobe.2018.04.001}{10.1016/j.anaerobe.2018.04.001}.

\leavevmode\hypertarget{ref-Jenior2017}{}%
31. \textbf{Jenior ML}, \textbf{Leslie JL}, \textbf{Young VB},
\textbf{Schloss PD}. 2017. \emph{Clostridium difficile} colonizes
alternative nutrient niches during infection across distinct murine gut
microbiomes. mSystems \textbf{2}.
doi:\href{https://doi.org/10.1128/msystems.00063-17}{10.1128/msystems.00063-17}.

\leavevmode\hypertarget{ref-Jenior2018}{}%
32. \textbf{Jenior ML}, \textbf{Leslie JL}, \textbf{Young VB},
\textbf{Schloss PD}. 2018. \emph{Clostridium difficile} alters the
structure and metabolism of distinct cecal microbiomes during initial
infection to promote sustained colonization. mSphere \textbf{3}.
doi:\href{https://doi.org/10.1128/msphere.00261-18}{10.1128/msphere.00261-18}.

\leavevmode\hypertarget{ref-Maier2018}{}%
33. \textbf{Maier L}, \textbf{Pruteanu M}, \textbf{Kuhn M},
\textbf{Zeller G}, \textbf{Telzerow A}, \textbf{Anderson EE},
\textbf{Brochado AR}, \textbf{Fernandez KC}, \textbf{Dose H},
\textbf{Mori H}, \textbf{Patil KR}, \textbf{Bork P}, \textbf{Typas A}.
2018. Extensive impact of non-antibiotic drugs on human gut bacteria.
Nature \textbf{555}:623--628.
doi:\href{https://doi.org/10.1038/nature25979}{10.1038/nature25979}.

\leavevmode\hypertarget{ref-Rinttila2004}{}%
34. \textbf{Rinttila T}, \textbf{Kassinen A}, \textbf{Malinen E},
\textbf{Krogius L}, \textbf{Palva A}. 2004. Development of an extensive
set of 16S rDNA-targeted primers for quantification of pathogenic and
indigenous bacteria in faecal samples by real-time PCR. Journal of
Applied Microbiology \textbf{97}:1166--1177.
doi:\href{https://doi.org/10.1111/j.1365-2672.2004.02409.x}{10.1111/j.1365-2672.2004.02409.x}.

\leavevmode\hypertarget{ref-Sorg2009}{}%
35. \textbf{Sorg JA}, \textbf{Dineen SS}. 2009. Laboratory maintenance
of \emph{Clostridium difficile}. Current Protocols in Microbiology
\textbf{12}.
doi:\href{https://doi.org/10.1002/9780471729259.mc09a01s12}{10.1002/9780471729259.mc09a01s12}.

\leavevmode\hypertarget{ref-Winston2016}{}%
36. \textbf{Winston JA}, \textbf{Thanissery R}, \textbf{Montgomery SA},
\textbf{Theriot CM}. 2016. Cefoperazone-treated mouse model of
clinically-relevant \(\Delta\)\emph{Clostridium difficile} strain
r20291. Journal of Visualized Experiments.
doi:\href{https://doi.org/10.3791/54850}{10.3791/54850}.

\leavevmode\hypertarget{ref-Kozich2013}{}%
37. \textbf{Kozich JJ}, \textbf{Westcott SL}, \textbf{Baxter NT},
\textbf{Highlander SK}, \textbf{Schloss PD}. 2013. Development of a
dual-index sequencing strategy and curation pipeline for analyzing
amplicon sequence data on the MiSeq illumina sequencing platform.
Applied and Environmental Microbiology \textbf{79}:5112--5120.
doi:\href{https://doi.org/10.1128/aem.01043-13}{10.1128/aem.01043-13}.

\leavevmode\hypertarget{ref-Schloss2009}{}%
38. \textbf{Schloss PD}, \textbf{Westcott SL}, \textbf{Ryabin T},
\textbf{Hall JR}, \textbf{Hartmann M}, \textbf{Hollister EB},
\textbf{Lesniewski RA}, \textbf{Oakley BB}, \textbf{Parks DH},
\textbf{Robinson CJ}, \textbf{Sahl JW}, \textbf{Stres B},
\textbf{Thallinger GG}, \textbf{Horn DJV}, \textbf{Weber CF}. 2009.
Introducing mothur: Open-source, platform-independent,
community-supported software for describing and comparing microbial
communities. Applied and Environmental Microbiology
\textbf{75}:7537--7541.
doi:\href{https://doi.org/10.1128/aem.01541-09}{10.1128/aem.01541-09}.

\leavevmode\hypertarget{ref-Wang2007}{}%
39. \textbf{Wang Q}, \textbf{Garrity GM}, \textbf{Tiedje JM},
\textbf{Cole JR}. 2007. Na√Øve bayesian classifier for rapid assignment
of rRNA sequences into the new bacterial taxonomy. Applied and
Environmental Microbiology \textbf{73}:5261--5267.
doi:\href{https://doi.org/10.1128/aem.00062-07}{10.1128/aem.00062-07}.

\leavevmode\hypertarget{ref-Yue2005}{}%
40. \textbf{Yue JC}, \textbf{Clayton MK}. 2005. A similarity measure
based on species proportions. Communications in Statistics - Theory and
Methods \textbf{34}:2123--2131.
doi:\href{https://doi.org/10.1080/sta-200066418}{10.1080/sta-200066418}.

\leavevmode\hypertarget{ref-Segata2011}{}%
41. \textbf{Segata N}, \textbf{Izard J}, \textbf{Waldron L},
\textbf{Gevers D}, \textbf{Miropolsky L}, \textbf{Garrett WS},
\textbf{Huttenhower C}. 2011. Metagenomic biomarker discovery and
explanation. Genome Biology \textbf{12}:R60.
doi:\href{https://doi.org/10.1186/gb-2011-12-6-r60}{10.1186/gb-2011-12-6-r60}.

\leavevmode\hypertarget{ref-Benjamini1995}{}%
42. \textbf{Benjamini Y}, \textbf{Hochberg Y}. 1995. Controlling the
false discovery rate: A practical and powerful approach to multiple
testing. Journal of the Royal Statistical Society: Series B
(Methodological) \textbf{57}:289--300.
doi:\href{https://doi.org/10.1111/j.2517-6161.1995.tb02031.x}{10.1111/j.2517-6161.1995.tb02031.x}.

\leavevmode\hypertarget{ref-Kurtz2015}{}%
43. \textbf{Kurtz ZD}, \textbf{M√ºller CL}, \textbf{Miraldi ER},
\textbf{Littman DR}, \textbf{Blaser MJ}, \textbf{Bonneau RA}. 2015.
Sparse and compositionally robust inference of microbial ecological
networks. PLOS Computational Biology \textbf{11}:e1004226.
doi:\href{https://doi.org/10.1371/journal.pcbi.1004226}{10.1371/journal.pcbi.1004226}.
\end{cslreferences}

\newpage

\textbf{Figure 1. Mouse experiment timeline.} Mice were given water with
cefoperazone (0.5 mg/ml) or streptomycin (5 mg/ml) for 5 days. The mice
were returned to untreated water for the remainder of the experiment.
Two days after the antibiotic water was removed, mice were orally
gavaged 100 \(\mu\)l of PBS or fecal community, once a day for two days.
The following day, the mice were challenged with 10\textsuperscript{3}
\emph{C. difficile} 630 spores. Alternatively, mice were given an
intraperitoneal injection of clindamycin (10 mg/kg) 2 days prior to
\emph{C. difficile} infection. 24 hours later, mice were orally gavaged
with 100 \(\mu\)l of PBS or fecal community. The following day, the mice
were challenged with 10\textsuperscript{3} \emph{C. difficile} 630
spores. Fecal pellets were collected prior to treatment (day -9 for
cefoperazone/streptomycin, day -2 for clindamycin), cessation of
antibiotics (day -2 for cefoperazone/streptomycin, day -1 for
clindamycin), prior to \emph{C. difficile} infection (day 0), and each
of the following 10 days.

\hfill\break

\textbf{Figure 2. Fecal community transplant inhibited \emph{C.
difficile} colonization for mice treated with cefoperazone or
streptomycin.} \emph{C. difficile} CFU per gram of feces for mice
treated with clindamycin (red points), cefoperazone (blue points), or
streptomycin (orange points). Mice were orally gavaged either PBS (open
circles) or FCT (fecal community transplant, filled circles) prior to
the \emph{C. difficile} infection. Each point represents an individual
mouse (Clindamycin - PBS n = 4, FCT n = 7; Cefoperazone - PBS n = 2, FCT
n = 2; Streptomycin - PBS n = 10, FCT n = 14). LOD = limit of detection.

\hfill\break

\textbf{Figure 3. Diluted FCT inhibited \emph{C. difficile} colonization
for mice treated with streptomycin.} \emph{C. difficile} CFU per gram of
feces for mice treated with cefoperazone (blue points) or streptomycin
(orange points). Mice were orally gavaged with a dilution of FCT (1:10
to 1:10\textsuperscript{5}) prior to the \emph{C. difficile} infection
at (A) one day post \emph{C. difficile} infection (dpi) and (B) 10 dpi.
Each point represents an individual mouse (Cefoperazone - 1:10 n = 2,
1:10\textsuperscript{2} n = 2, 1:10\textsuperscript{3} n = 3,
1:10\textsuperscript{4} n = 2, 1:10\textsuperscript{5} n = 2;
Streptomycin - 1:10 n = 12, 1:10\textsuperscript{2} n = 14,
1:10\textsuperscript{3} n = 5, 1:10\textsuperscript{4} n = 4,
1:10\textsuperscript{5} n = 5). LOD = limit of detection.

\hfill\break

\textbf{Figure 4. Diversity of murine gut bacterial community had not
recovered at the time of \emph{C. difficile} infection in
streptomycin-treated mice.} \(\alpha\)-diversity, measured by
S\textsubscript{obs} (A) and Inverse Simpson (B), prior to beginning
antibiotic treatment (day -9), before fecal community transplant (day
-2), after fecal community transplant on the day of \emph{C. difficile}
challenge (day 0) and at the end of the experiment (day 10). (C)
\(\beta\)-diversity, measured by \(\theta\)\textsubscript{YC}, distance
between community structures on day 0 or 10 compared to the community
prior to antibiotic treatment (day -9) community of that individual.
Data are grouped by the transplant received, undiluted fecal community
(FCT), diluted fecal community
(1:10\textsuperscript{1}-1:10\textsuperscript{5}), or PBS. Points are
median values and lines represent the interquartile range.

\hfill\break

\textbf{Figure 5. Bacterial community OTUs differentially abundant in
streptomycin-treated mice which resisted or cleared colonization.}
Murine gut bacterial community OTUs that were significantly different by
LEfSe analysis. OTUs from streptomycin-treated mice at the time of
\emph{C. difficile} challenge (day 0) which were differentially abundant
between (A) mice that were colonized (dark green) and those that were
not (no detectable CFU throughout the experiment, bright green) or (B)
mice that remained colonized (dark green) and those that cleared
colonization (CFU reduced to below the limit of detection by the end of
the experiment, faint green). (C) OTUs from streptomycin-treated mice at
the end of the experiment (day 10) which were differentially abundant
between mice that remained colonized (dark green) and those that cleared
colonization (CFU reduced to below the limit of detection by the end of
the experiment, faint green). Points are median values and lines
represent the interquartile range. Dashed vertical line is the limit of
detection. OTUs ordered alphabetically. * indicates that the OTU was
unclassified at lower classification rank.

\hfill\break

\textbf{Figure 6. Streptomycin-treated murine fecal community
associations with \emph{C. difficile}.} Network constructed with
SpiecEasi from the OTU relative abundances and \emph{C. difficile} CFU
data from 1 through 5 days post \emph{C. difficile} infection. Red lines
represent negative associations and blue lines indicate positive
associations. \emph{C. difficile} is based on CFU counts and
\emph{Peptostreptococcaceae} (OTU 19), the OTU most closely related to
\emph{C. difficile}, is based on sequence counts. Only \emph{C.
difficile} subnetwork shown.

\hfill\break

\textbf{Figure S1. \emph{C. difficile} colonization dynamics in
streptomycin-treated mice across all prophylactic transplant
treatments.} \emph{C. difficile} CFU per gram of feces for
streptomycin-treated mice orally gavaged PBS, fecal community transplant
(FCT), or diluted FCT (1:10-1:10\textsuperscript{5}) prior to the
\emph{C. difficile} infection. Each semi-transparent line represents an
individual mouse. Mice challenged with 10\textsuperscript{3} \emph{C.
difficile} 630 spores on day 0. Lines grouped by the transplant
treatment received. LOD = limit of detection.

\hfill\break

\textbf{Figure S2. Diversity and quantification of fecal community
dilutions used for prophylactic transplants in antibiotic-treated mice.}
(A-C) Diversity of fecal community dilutions. \(\alpha\)-diversity,
measured by (A) S\textsubscript{obs} and (B) Inverse Simpson for
undiluted fecal community (FCT) and diluted fecal communities
(1:10-1:10\textsuperscript{5}). Points are individual samples. (C)
\(\beta\)-diversity, measured by \(\theta\)\textsubscript{YC}, community
structure of feces collected from untreated mice, undiluted fecal
community (FCT), and diluted fecal communities
(1:10-1:10\textsuperscript{5}) compared to untreated feces. Points are
median values and lines represent the interquartile range. (D) Cq values
for qPCR of FCT and its dilutions for eubacterial 16S rRNA gene. Points
are median values and lines represent the interquartile range. (E)
Relative abundance of bacterial taxonomic groups that significantly
correlate with fecal community dilutions (FCT-1:10\textsuperscript{3})
by Spearman correlation. Points are individual mice. * indicates that
the bacterial taxonomic group was unclassified at lower classification
rank.

\hfill\break

\textbf{Figure S3. Diversity of murine gut bacterial community was not
recovered at the time of \emph{C. difficile} infection in
cefoperazone-treated mice.} Diversity changes through experiments with
cefoperazone-treated mice. \(\alpha\)-diversity, measured by (A)
S\textsubscript{obs} and (B) Inverse Simpson, prior to beginning
antibiotic treatment (day -9), before fecal community transplant (day
-2), after fecal community transplant on the day of \emph{C. difficile}
infection (day 0) and at the end of the experiment (day 10). (C)
\(\beta\)-diversity, measured by \(\theta\)\textsubscript{YC}, distance
between community structures on day 0 or 10 compared to the community
prior to antibiotic treatment (day -9) community of that individual.
Data are grouped by the transplant received, undiluted fecal community
(FCT), diluted fecal community
(1:10\textsuperscript{1}-1:10\textsuperscript{5}), or PBS. Points are
individual mice.

\hfill\break

\textbf{Figure S4. Gut bacterial community of streptomycin-treated mice
that cleared colonization were more similar to their initial community.}
Diversity differences by outcome in streptomycin-treated mice.
\(\beta\)-diversity, measured by \(\theta\)\textsubscript{YC}, distance
between community structures on day 0 or 10 compared to the community
prior to antibiotic treatment (day -9) community of that individual.
Data are grouped by the outcome, cleared colonization (faint green) or
remain colonized (dark green). Points are median values and lines
represent the interquartile range. * indicates significant difference by
Wilcoxon rank sum test with Bonferroni correction.

\hfill\break

\textbf{Figure S5. Murine gut bacterial community OTUs differentially
abundant with streptomycin treatment.} Murine gut bacterial community
OTUs that were significantly different by LEfSe analysis between
untreated mice (Initial, black) and after 5 days of water with
streptomycin (5 mg/ml) and 2 days of untreated water (After
streptomycin, orange). Large bold points represent the group median.
Small, semi-transparent points represent an individual mouse. Gray arrow
indicates the direction the relative abundance shifted with the
streptomycin treatment. Left plot displays OTUs with a median relative
abundance greater than 0.1\%, the OTUs lower are displayed in the right
plot. Dashed vertical line is the limit of detection. OTUs ordered
alphabetically. * indicates that the OTU was unclassified at lower
classification rank.

\hfill\break

\textbf{Figure S6. Murine gut bacterial community OTUs of
cefoperazone-treated mice at the time of challenge.} Murine gut
bacterial community OTUs that were present in at least one sample at the
time of \emph{C. difficile} challenge (day 0). Mice were pre-treated
with either fecal community transplant (FCT, open circles) or FCT
diluted 1:10 (filled circles). Points are individual samples. Dashed
vertical line is the limit of detection. OTUs ordered alphabetically. *
indicates that the OTU was unclassified at lower classification rank.

\end{document}
